src/sql/workbench/parts/notebook/notebook.component.ts

```diff
@@ -582,4 +582,51 @@ export class NotebookComponent extends AngularDisposable implements OnInit, OnDe
 		}
 	}
 
+	getSections(): INotebookSection[] {
+		return this.getSectionElements();
```

Comment:
i'm curious about the rationale for the wrapper here

Reply:
I just didn't want to expose the NotebookSection class (with HTML Element) publicly, but didn't want to have to do a cast when using in the other method.